command.go

```diff
@@ -512,18 +512,20 @@ func (cmds CommandSet) Call(ctx context.Context, args, env []string) (int, error
 		return 0, &Help{Cmd: cmds}
 	}
 
-	if len(args) == 0 {
-		return 1, &Usage{Cmd: cmds, Err: fmt.Errorf("missing command")}
+	var a string
+	var otherargs []string
+	if len(args) > 0 {
+		a = args[0]
+		otherargs = args[1:]
 	}
 
-	a := args[0]
 	c := cmds[a]
 
 	if c == nil {
 		return 1, &Usage{Cmd: cmds, Err: fmt.Errorf("unknown command: %q", a)}
 	}
 
-	return NamedCommand(a, c).Call(ctx, args[1:], env)
+	return NamedCommand(a, c).Call(ctx, otherargs, env)
```

Comment:
Does this still error if no empty command exist in the CommandSet?

Reply:
It does not, and this also gets you a weird failure if you include other flags/args in your command, so this is pretty brittle atm. I'll add some other test cases here to make sure we still error correctly if the command is not found entirely.